controllers/velero.go

```diff
@@ -144,6 +146,28 @@ func (r *DPAReconciler) veleroClusterRoleBinding(dpa *oadpv1alpha1.DataProtectio
 	return crb, nil
 }
 
+// Apply Override unsupported Args
+func (r *DPAReconciler) applyUnsupportedArgsOverride(dpa *oadpv1alpha1.DataProtectionApplication, veleroDeployment *appsv1.Deployment, veleroContainer *corev1.Container) error {
+	if configMapName, ok := dpa.Annotations[UnsupportedServerArgsAnnotation]; ok {
+		// Got an DPA annotation for Unsupported Server Args
+		if configMapName == "" {
+			// Can't be empty string, do not report error
+			return nil
+		}
+
+		unsupportedServerArgsCM := corev1.ConfigMap{}
+		if err := r.Get(r.Context, types.NamespacedName{Namespace: r.NamespacedName.Namespace, Name: configMapName}, &unsupportedServerArgsCM); err == nil {
```

Comment:
would change order here to remove `else`

```go
if err := r.Get(r.Context, types.NamespacedName{Namespace: r.NamespacedName.Namespace, Name: configMapName}, &unsupportedServerArgsCM); err != nil {
    return err
}
r.Log.Info("Applying Unsupported Server Args.")
// if server args is set, override the default server args
veleroContainer.Args = common.GenerateCliArgsFromConfigMap("server", &unsupportedServerArgsCM)
```

Reply:
thanks, much simpler